docs/acme-divergences.md

```diff
@@ -2,9 +2,11 @@
 
 While Boulder attempts to implement the ACME specification as strictly as possible there are places at which we will diverge from the letter of the specification for various reasons.
 
-This document details these differences, since ACME is not yet finalized it will be updated as numbered drafts are published.
+Since Boulder evolved alongside the ACME specification there is not one exact ACME draft number that can be referenced in isolation to understand Boulder. This document exists to detail differences between what Boulder does and the most-recently published ACME draft. Since ACME is not yet finalized it will be updated as new numbered drafts are published.
```

Comment:
Perhaps instead of `to understand Boulder` it'd be slightly clearer as `to understand the protocol Boulder speaks`?

Reply:
Sounds good - changed in 16cd273 Thanks!